### PR TITLE
Fix msql migrations

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- Essentially: mysql needs table_schema set when querying if a column exists.

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-31140

### CODE REVIEW

**Review by lieut-data — COMMENTED** [review]
Quick comment: is it worth just querying `SELECT DATABASE();` when needed vs. the signature change?

**Comment by cpoile** [comment]
@lieut-data 🤦 I feel like an idiot. That's much easier.

File: server/sqlstore/migrations_utils.go [hunk 1]
```diff
@@ -39,9 +39,16 @@ var addColumnToPGTable = func(e sqlx.Ext, tableName, columnName, columnType stri
...
+	err := e.QueryRowx("SELECT DATABASE();").Scan(&dbName)
+	if err != nil {
+		return err
+	}
+
 	var result int
-	err := e.QueryRowx(
-		"SELECT 1 FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME = ? AND COLUMN_NAME = ?",
+	err = e.QueryRowx(
+		"SELECT 1 FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_SCHEMA = ? AND TABLE_NAME = ? AND COLUMN_NAME = ?",
```
Comment (lieut-data) [inline server/sqlstore/migrations_utils.go:50]: Could also inline this as `TABLE_SCHEMA = DATABASE()` to avoid the extra query.

**Review by lieut-data — APPROVED** [review]
Thanks for getting to the root problem either way, @cpoile :)

**Review by crspeller — APPROVED** [review]

**Pull request merged**